src/components/RightHandMenu/DonutChart/UnEqualDonut/UnequalDonut.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
Open to different ideas here, but questions:
1. What is the distinction we are making between UnEqualDonut and regular Donut?
2. Would it make more sense to have one component that had different props passed in?

Reply:
We have 2 different types of pie charts.... 
-one with just 2 data (width of 1 section is less than other)
-another with many data (race type)

Initially tried to make both as 1 component and just change the styling based of props, but turns out it's 2 different ways to create them.

Reply:
Okay, let's keep it. Thanks for explaining!